views/layout.jade

```diff
@@ -20,6 +20,7 @@ html
     script(src='https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js')
     script(src='/javascripts/bootstrap.js')
     script(src='/javascripts/nfl-data.js')
+    script NFLData.USE_CACHED_PLAYER_DATA = #{USE_CACHED_PLAYER_DATA || false};
```

Comment:
I think down the road the best way would be to have the server get the player data, and serve it up to the client. Then only the server needs to know whether the nfl data is cached or not. This is good for now though.